src/main/java/com/pubnub/api/models/consumer/objects_vsp/user/User.java

```diff
@@ -0,0 +1,31 @@
+package com.pubnub.api.models.consumer.objects_vsp.user;
+
+import com.pubnub.api.models.consumer.objects_api.PNObject;
+import lombok.*;
+import lombok.experimental.Accessors;
+
+@Getter
+@Setter
+@Accessors(chain = true)
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+@NoArgsConstructor
+public class User extends PNObject {
```

Comment:
Same story as with Space. I think its `id` should have type `UserId`

Reply:
I will add it. Thanks.